proxy-node-shared/src/main/java/uk/gov/ida/notification/saml/EidasAssertionBuilder.java

```diff
@@ -64,12 +66,24 @@ public Assertion build() {
         return assertion;
     }
 
-    private Subject createSubject(String pid) {
+    private Subject createSubject(String pid, String requestId, String destination) {
         Subject subject = SamlBuilder.build(Subject.DEFAULT_ELEMENT_NAME);
         NameID nameID = SamlBuilder.build(NameID.DEFAULT_ELEMENT_NAME);
         nameID.setValue(pid);
         nameID.setFormat(NameIDType.PERSISTENT);
         subject.setNameID(nameID);
+
+        SubjectConfirmationData subjectConfirmationData = SamlBuilder.build(SubjectConfirmationData.DEFAULT_ELEMENT_NAME);
+        subjectConfirmationData.setInResponseTo(requestId);
+        subjectConfirmationData.setNotBefore(DateTime.now());
+        subjectConfirmationData.setNotOnOrAfter(DateTime.now().plusMinutes(5));
+        subjectConfirmationData.setRecipient(destination);
+
+        SubjectConfirmation subjectConfirmation = SamlBuilder.build(SubjectConfirmation.DEFAULT_ELEMENT_NAME);
+        subjectConfirmation.setMethod("urn:oasis:names:tc:saml2:2.0:cm:bearer");
```

Comment:
We should the constant provided by OpenSAML here
`SubjectConfirmation.METHOD_BEARER`